### DISCUSSION TITLE
XMLSec 1.3.4-rc1

### DISCUSSION THREAD

**Discussion by lsh123** [lsh123/xmlsec#774] [Announcements]
The **XML Security Library 1.3.4** release includes the following changes:

- (xmlsec-openssl) Support cert dates before unix epoch start.
- (xmlsec-openssl) Fix build for LibreSSL or BoringSSL.
- (xmlsec-nss) Ensure NSS algorithms are initialized.
- Several other small fixes (see [more details](https://github.com/lsh123/xmlsec/commits/master)).

Please test the release [candidate](https://www.aleksey.com/xmlsec/download/xmlsec1-1.3.4-rc1.tar.gz) ([signature](https://www.aleksey.com/xmlsec/download/xmlsec1-1.3.4-rc1.sig)) and let me know if you see any issues!

<hr /><em>This discussion was created from the release <a href='https://github.com/lsh123/xmlsec/releases/tag/1.3.4-rc1'>XMLSec 1.3.4-rc1</a>.</em>

**Comment by vmiklos** [comment]
I launched some odfsig integration tests in general + libreoffice linux integration tests, looks OK so far.

**Comment by lsh123** [comment]
Thank you!
